dds_web/api/user.py

```diff
@@ -449,6 +461,12 @@ def delete(self):
         msg = f"The user account {user.username} ({user_email_str}, {user.role})  has been terminated successfully been by {current_user.name} ({current_user.role})."
         flask.current_app.logger.info(msg)
 
+        with structlog.threadlocal.bound_threadlocal(
+            who={"user": user.username, "role": user.role},
+            by_whom={"user": current_user.username, "role": current_user.role},
+        ):
+            action_logger.info(self.__class__)
+
```

Comment:
Do we need this now that we have the decorator?

Reply:
Not necessarily. It depends on how verbose you as Super Admin would like the logs to be.

The basic idea was that the decorator would only cover the bare minimum of information that is required for each log entry and that additional logs can be written for important actions inside the respective functions.

In this case, I felt that user deletion is such a relevant action, that one might want to access additional information quickly without studding it from the request and username fields. Having the separate, function-specific JSON fields in the logs would simplify aggregating and accessing this information later with `jq`.

Reply:
Okey! Good point, lets keep this the way it is.